document-store/src/main/java/org/hypertrace/core/documentstore/mongo/query/parser/MongoGroupTypeExpressionParser.java

```diff
@@ -99,4 +128,15 @@ private Map<String, Object> parse(final GroupTypeExpression expression) {
     MongoGroupTypeExpressionParser parser = new MongoGroupTypeExpressionParser();
     return expression.accept(parser);
   }
+
+  private static List<SelectionSpec> getFunctionExpressionSelectionWithGroupBys(
```

Comment:
Where is this getting used?

Reply:
https://github.com/hypertrace/document-store/blob/93fdf172a73d66625ccf0b59dbdc2736cddb47ca/document-store/src/main/java/org/hypertrace/core/documentstore/mongo/query/parser/MongoGroupTypeExpressionParser.java#L56